### ISSUE TITLE
Add transformation functions

### ISSUE THREAD

**Issue by vsoch** [vsoch/pybraincompare#13]
The goal of this enhancement is to allow for synchronizing different applications that both calculate transformations (eg, NeuroVault and NeuroSynth). Specifically, in the words of Chris:

1. creating, reading and writing reduced representations of a brain image. This is a vector of values created by downsampling the map to 4x4x4, reslicing it and picking only the voxels from within the brain mask. We use this representation for similarity calculations. It's close to what you do for decoding (using random sampling)

2. calculating similarity between two maps - it's fairly simple, (it's just a correlation with pairwise deletion), but we can also share the code for it

In summary - shared downsampling code and low-resolution image representation.

**Comment by tyarkoni** [comment]
Sounds good. Let me know if you need me to do anything, but as long as the downsampling function takes an image as input and returns a numpy array as output, I should be able to inject it into my pipeline without any problems.

**Comment by vsoch** [comment]
Thanks Tal. I'll let you know when it's ready for testing, and I will most definitely want your feedback.  Specifically, on this point:

     I do some optimization during preprocessing (specifically, I standardize all images before writing them to the memmap, which lets me do a single np.dot() to get all correlation coefficients.)

**Comment by vsoch** [comment]
Ok, preliminary [functions are here](https://github.com/vsoch/pybraincompare/blob/update/tranformations/pybraincompare/mr/transformation.py). I tested manually, and am next going to do some real tests (or @chrisfilo will tar and feather me :))

**Comment by vsoch** [comment]
Added and tested with https://github.com/vsoch/pybraincompare/pull/14
